packages/noya-designsystem/src/components/DropdownMenu.tsx

```diff
@@ -148,6 +152,7 @@ function DropdownMenuRoot<T extends string>({
         </RadixDropdownMenu.Trigger>
       )}
       <RootElement
+        style={style}
```

Comment:
Instead of passing a style here, let's pass the `scrollable` prop into this component, and handle a `scrollable` prop here in the `contentStyle` function that's shared by both context menus and dropdown menus:

https://github.com/noya-app/noya/blob/9cfbf15329ab6dd103a3a3429de5a183d07337d0/packages/noya-designsystem/src/components/internal/Menu.tsx#L82

We pass `contentStyle` when setting up the `RootElement`: https://github.com/noya-app/noya/blob/9cfbf15329ab6dd103a3a3429de5a183d07337d0/packages/noya-designsystem/src/components/DropdownMenu.tsx#L116